diff --git a/dist/compiler/utils/postcss.js b/dist/compiler/utils/postcss.js
index 85a9ce5586dc886a035dca9842e014b8cd43cabb..652ad50f08c5ed85bea7c2cbb004b75cae3ba421 100644
--- a/dist/compiler/utils/postcss.js
+++ b/dist/compiler/utils/postcss.js
@@ -39,12 +39,15 @@ async function loadPostcssPlugins({
   config,
   context = defaultContext
 }) {
+  console.log(config)
   if (!isPostcssEnabled(config)) {
+    console.log("postcss disabled")
     return [];
   }
   let {
     rootDirectory
   } = config;
+  console.log("rootDirectory", rootDirectory)
   let cacheKey = getCacheKey({
     config,
     context
@@ -55,6 +58,7 @@ async function loadPostcssPlugins({
   }
   let plugins = [];
   if (config.postcss) {
+    console.log("config.postcss", config.postcss)
     try {
       let postcssConfig = await loadConfig__default["default"](
       // We're nesting our custom context values in a "remix"
@@ -63,8 +67,10 @@ async function loadPostcssPlugins({
       {
         remix: context
       }, rootDirectory);
+      console.log("postcssConfig", postcssConfig)
       plugins.push(...postcssConfig.plugins);
     } catch (err) {
+      console.error(err)
       // If they don't have a PostCSS config, just ignore it.
     }
   }
