diff --git a/dist/index.cjs b/dist/index.cjs
index c64a04b739be64f6b9d59264308b2a2f20df5b2e..a46d3cae0113af4b4cb3d9bba266dd588eac8907 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -61,7 +61,12 @@ const order = utils.ESLintUtils.RuleCreator((name) => name)({
           if (node.value?.[0].type === "SvelteLiteral")
             checkLiteral(node.value[0]);
         }
-      }
+      },
+      // check plain strings that start with :uno:
+      Literal(node) {
+        if (typeof node.value === "string" && node.value.startsWith(":uno:"))
+          checkLiteral(node);
+      },
     };
     const templateBodyVisitor = {
       VAttribute(node) {
